src/phases/execution/ClusteredCollection.yml

```diff
@@ -134,17 +146,76 @@ Actors:
         Operations:
         - OperationName: find
           OperationCommand:
-            Filter: {a: {^FastRandomString: {length: 6, alphabet: "0123456789"}}}
+            Filter: {a: *RandomSecondaryKey}
 
+# Phase 6: Range lookup with both primary and secondary key. Primary key predicate is more selective.
+- Name: RangeLookupBySelectivePrimaryAndWideSecondaryIndexes
```

Comment:
Personally, I would find it helpful to clarify what being more selective or wide means.

Reply:
I re-wrote the comments and stage names to be more readable.